mandatory/3-playing-computer.js

```diff
@@ -28,15 +29,16 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
-console.log(b);
+
+//console.log(b);  // b is not defined
 
 for (let i = 0; i < 5; ++i) {
   a = a + 1;
   if (i % 2 === 0) {
-    const d = f2(i, x);
+    const d = f2(i, x); //f2 called once
     console.log(d);
   } else {
-    const e = f1(i, a);
+    const e = f1(i, a); //f1 called once, a takes a value of i
```

Comment:
Good work krishan, for the mandatory exercise number 3. Maybe have a look at what is returned to the console when you run the code and then have a think about how many times the f1 and f2 functions are called? Remember that the functions are both inside the for loop which runs from i=0 to i<5.

Reply:
Thanks Peter for reviewing it! I will look into this man 👍🏻